packages/lib-classifier/src/hooks/useFreehandLineReductions.js

```diff
@@ -2,11 +2,8 @@ import { getType } from 'mobx-state-tree'
 import { useStores } from '@hooks'
 import { useCaesarReductions } from './'
 
-export default function useFreehandLineReductions() {
+export default function useFreehandLineReductions(frame = 0) {
```

Comment:
I think this is a way to filter reductions, but I don't totally understand what `useFreehandLineReductions` accomplishes. See questions in https://github.com/zooniverse/front-end-monorepo/pull/5754#pullrequestreview-1822355926